app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,18 @@
+class NotificationsController < ApplicationController
+  
+  def create
+    notification = Notification.new(notification_params)
+    message = {notification_status: "success"}
+    unless notification.save
+      message = {notification_status: "not_success"}
```

Comment:
This is weird. You assign success then reassign in an unless. Just use if / else.